test/index.test.js

```diff
@@ -1,24 +1,13 @@
+/**
+ * @jest-environment: node
+ */
 var createStore = require('storeon')
 
 var router = require('../')
 
 /* eslint es5/no-rest-parameters:0, es5/no-arrow-functions:0 */
 /* eslint es5/no-shorthand-properties:0 */
 
-var clickOnBody = () => {}
-beforeAll(() => {
-  jest.spyOn(document.body, 'addEventListener')
-    .mockImplementation((event, callback) => {
-      if (event === 'click') {
-        clickOnBody = callback
-      }
-    })
-})
-
-beforeEach(() => {
-  history.pushState(null, null, '/')
-})
-
 it('init state', () => {
```

Comment:
Help me, please - why this test pass?

Reply:
@ai @isolovev

Reply:
Let’s first discuss this question https://github.com/storeon/router/pull/8#discussion_r291181006

Maybe with new API this error will be fixed automatically